.github/workflows/operator_release.yaml

```diff
@@ -93,4 +94,28 @@ jobs:
         gh release create "${TAG_NAME}" --title "Operator ${TAG_NAME#release/operator/}" \
           --notes $'Pixie Operator Release:\n'"${changelog}"
         gh release upload "${TAG_NAME}" operator-artifacts/*
+    # yamllint enable rule:indentation
```

Comment:
we should add `concurrency: gh-pages` so that only one job that touches github pages runs at a time. And if you want we could separate the pushing to gh-pages into a separate job, so that the concurrency only affects the push, and the build step can still run concurrently with other stuff.